skbio/stats/power.py

```diff
@@ -942,22 +1014,17 @@ def _calculate_power_curve(test, samples, sample_counts, ratio=None,
     num_iter : int
         The default is 1000. The number of p-values to generate for each point
         on the curve.
-
```

Comment:
I'm not sure if Sphinx/numpydoc cares, but probably good to add these lines back for readability (e.g., if a user is reading the help in an ipython session).